lisa/runners/lisa_runner.py

```diff
@@ -602,7 +604,7 @@ def _attach_failed_environment_to_result(
         # so deployment failure can be tracked.
         environment.platform = self.platform
         result.environment = environment
-
+        self._capture_vm_log_in_test_dir(environment, result)
```

Comment:
How about just copy files from env log folder to the test case folder? It will be consistent and reduce calls to get serial log and so on.

Reply:
Typically, there would be around 5-6 serial console log folders within the environment folder.
Do you recommend copying all of those rather than the last serial console folder?

Reply:
Yes. Copy it will be more stable and faster. It also provides more information like arm templates or other extra log files.